segmentstore/contracts/src/main/java/io/pravega/segmentstore/contracts/Attributes.java

```diff
@@ -125,6 +125,19 @@
      */
     public static final AttributeId ATTRIBUTE_ID_LENGTH = AttributeId.uuid(CORE_ATTRIBUTE_ID_PREFIX, 12);
 
+    /**
+     * Defines an attribute that is used to store the BatchId for a batch of segments to be merged.
+     * If not specified (or if set to 0), the default will be the length of {@link AttributeId.UUID} (16 bytes).
+     */
+    public static final AttributeId SEG_MERGE_BATCH_ID = AttributeId.uuid(CORE_ATTRIBUTE_ID_PREFIX, 13);
```

Comment:
I would remove `SEG_`. All of these attributes pertain to Segments, so no need to add that.

Care must be taken when merging this in. @co-jo let's fix your attribute in #6171 to be 15; otherwise we will have conflicting attribute ids (and we won't even know it as the file merge will go just fine)

Reply:
fixed